README.md

```diff
@@ -108,18 +108,30 @@ Option 1: Pull down the image from [Docker Hub](https://hub.docker.com/r/qasimwa
 
 `docker pull qasimwarraich/cli-tutor`
 
+`podman pull qasimwarraich/cli-tutor`
+
 Note: You can run the container using:
+
 `docker run -it qasimwarraich/cli-tutor`
 
+`podman run -it qasimwarraich/cli-tutor`
+
 ---
 
 Option 2: Building image without make
```

Comment:
Speaking of make, the `Makefile` should also reflect the change to `podman`.

Reply:
Want me to split up the make changes in another seperate commit/PR?